src/app/screens/Accounts.tsx

```diff
@@ -0,0 +1,126 @@
+import { useState, useEffect } from "react";
+import { WalletIcon } from "@bitcoin-design/bitcoin-icons-react/outline";
+
+import type { Accounts } from "../../types";
+import utils from "../../common/lib/utils";
+import { removeAccountFromCache } from "../../common/lib/cache";
+import { useAuth } from "../context/AuthContext";
+
+import Badge from "../components/Badge";
+import Container from "../components/Container";
+
+function AccountsScreen() {
+  const [accounts, setAccounts] = useState<Accounts>({});
+  const auth = useAuth();
+
+  function getAccounts() {
+    utils.call<Accounts>("getAccounts").then((response) => {
+      setAccounts(response);
+    });
+  }
+
+  async function selectAccount(accountId: string) {
+    auth.setAccountId(accountId);
+    await utils.call("selectAccount", {
+      id: accountId,
+    });
+    auth.fetchAccountInfo(accountId);
+  }
+
+  async function removeAccount(id: string, name: string) {
+    if (window.confirm(`Are you sure you want to delete account: ${name}?`)) {
+      let nextAccountId;
+      let accountIds = Object.keys(accounts);
+      if (auth.account?.id === id && accountIds.length > 1) {
+        nextAccountId = accountIds.filter((accountId) => accountId !== id)[0];
+      }
+
+      await Promise.all([
+        utils.call("deleteAccount", { id }),
+        removeAccountFromCache(id), // Clear cached account info (alias/balance).
```

Comment:
ahhh.. I hate cache invalidation... :D

can we do this in the api? and use `api.deleteAccount()` instead of `utils.call("deleteAccount"...)?